### ISSUE TITLE
Security alert for redcarpet gem

### ISSUE THREAD

**Issue by ElliotFriend** [github/pages-gem#134]
I'm told by gemnasium that there's a critical security fix contained in redcarpet 3.2.3. However, github-pages has that gem locked at 3.1.2. If possible, that gem should be updated as quickly as possible. More information available [on osvdb](http://osvdb.org/show/osvdb/120415) or [on gemnasium](https://gemnasium.com/gems/redcarpet/versions/3.2.3).

Would I be right in thinking that if a site is using kramdown, that there's some protection from this security vulnerability?

**Comment by Stargator** [comment]
I was just coming here to post the same thing. I didn't know kramdown might provide any protection for it. If a newer version fixes the issue, then I would always try to update, unless there are bigger issues that prevent that (such as broken features, budget, time, etc).

Any update on this from github? I forked the project to try and test pages-gem if all the dependencies were updated, but since I'm on Windows my testing has not been consistent. I'll have to wait until my Linux machine is ready before I can take another stab.
